go/enclave/db/storage.go

```diff
@@ -225,20 +225,20 @@ func (s *storageImpl) Proof(r *core.Rollup) *types.Block {
 	return v
 }
 
-func (s *storageImpl) FetchBlockState(hash common.L1RootHash) (*core.BlockState, bool) {
-	bs := obscurorawdb.ReadBlockState(s.db, hash, s.logger)
-	if bs != nil {
-		return bs, true
+func (s *storageImpl) FetchBlockState(hash common.L1RootHash) (*core.BlockState, error) {
+	bs, err := obscurorawdb.ReadBlockState(s.db, hash)
```

Comment:
Sugestion `return obscurorawdb.ReadBlockState(s.db, hash)` - no need to handle the error if nothing is handled